tests/test_bar.py

```diff
@@ -180,6 +180,11 @@ def test_df_from_u_kln_does_not_raise_on_incomplete_convergence():
     assert np.isfinite(df)
     assert np.isnan(ddf)  # returns NaN for uncertainty on incomplete convergence
 
+    bootstrap_df, bootstrap_ddf = bar_with_pessimistic_uncertainty(u_kln, maximum_iterations=1)
```

Comment:
This raises an exception without the fix. Didn't launch a CI job as it is an unambigious failure